### DISCUSSION TITLE
Is there a volume option in the app?

### DISCUSSION THREAD

**Discussion by pixeltrain3D** [Suplanus/Cuterdio#203] [General] [answered]
As the title says, can I change the output volume inside the app somewhere?
I would love to hear music while working, which involves a lot of watching parts of tutorials. In other music apps I lower th volume of the music to have it only gentle in the background.
Haven't found a volume in the app ...

Thanks a lot,
Helge

**Comment by Suplanus** [comment, marked as answer]
Swip from left to right and in the menu you have the volume + Airplay control.

**Reply by pixeltrain3D** [reply]
I'm on the desktop. So, I found, that the menu is much longer than the text items there, sorry!
Swipe is than on the mobile app, right?
Great app !

**Reply by Suplanus** [reply]
Swipe is on mobile right.
And yes. On Desktop its also in the menu.
